clamav-mirror/Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM python:3.9.9-alpine3.13
+FROM python:3.10.6-alpine3.15
 
-ENV CLAM_VERSION=0.103.6-r0
+ENV CLAM_VERSION=0.104.3-r0
 ENV SUPERCRONIC_URL=https://github.com/aptible/supercronic/releases/download/v0.1.12/supercronic-linux-amd64 \
```

Comment:
This is quite old (https://github.com/aptible/supercronic/releases/tag/v0.2.1)

Reply:
updated